projects/js-packages/components/components/status/style.module.scss

```diff
@@ -0,0 +1,63 @@
+.status {
+    --font-size: var( --font-body-extra-small ); // 12px
+    --indicator-size: calc( var( --font-size ) * 0.666 ); // 8px
+
+    align-items: center;
+    display: flex;
+    font-size: var( --font-size );
+    font-weight: 600;
+    line-height: 1.666;
+    white-space: nowrap;
+
+    &__indicator {
+        background-color: var( --jp-gray-50 );
+        border-radius: var( --indicator-size );
+        flex-shrink: 0;
+        height: var( --indicator-size );
+        width: var( --indicator-size );
```

Comment:
If you want to use it appended to `font-size,` why to not use `em`? Instead doing the math

Reply:
Good point! Resolved via https://github.com/Automattic/jetpack/pull/30209/commits/0b7c845d0def850fe5099d654fe2371ad25a17b2